src/apps/frontend/components/ellipsis/ellipsis-menu.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+
+import TaskModal from '../task/task-modal';
+
+const EllipsisMenu: React.FC = () => {
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+  const [modalOpen, setModalOpen] = useState(false);
+
+  return (
+    <div className="relative flex">
+      <button
+        className="text-[#98A6AD] hover:text-body"
```

Comment:
`text-[#98A6AD]` is not correct. Please declare the Color code in `tailwind.config.cjs`